webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/SystemProperties/index.js

```diff
@@ -0,0 +1,214 @@
+      </DescriptionListGroup>
+      <Slot id="host-details-tab-properties-1" multi />
+    </DescriptionList>
+    <Divider className="padded-divider" />
+    <DescriptionList
+      isCompact
+      columnModifier={{
+        default: '2Col',
+      }}
+    >
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Model')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {model}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Host group')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {hostgroupName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {ownerName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner type')}</DescriptionListTerm>
+        <DescriptionListDescription>{ownerType}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Location')}</DescriptionListTerm>
+        <DescriptionListDescription>{location}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Organization')}</DescriptionListTerm>
+        <DescriptionListDescription>{organization}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <Slot id="host-details-tab-properties-2" multi />
+    </DescriptionList>
+    <Divider className="padded-divider" />
+    <DescriptionList isCompact>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Created at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {createdAt && <LongDateTime date={createdAt} />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Updated at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {updateAt && <LongDateTime date={updateAt} />}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Uptime (s)')}</DescriptionListTerm>
```

Comment:
What does the `s` say? You're formatting it in something else than seconds.

Reply:
just a leftover :)